### PR TITLE
refactor(console,schemas,shared): extract dark mode color generator

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

extract dark mode color generator to `shared`, and use it in seed.

## Testing

pass existing UTs, and UI looks fine in local test.

### CODE REVIEW

**Review by gao-sun — CHANGES_REQUESTED** [review]
wrong lock file?

**Comment by wangsijie** [comment]
@gao-sun updated, thanks

File: packages/schemas/src/seeds/sign-in-experience.ts [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
+const defaultPrimaryColor = '#6139F6';
+
 export const defaultSignInExperience: Readonly<CreateSignInExperience> = {
   id: 'default',
   color: {
-    primaryColor: '#6139F6',
-    isDarkModeEnabled: false,
-    darkPrimaryColor: '#8768F8',
+    primaryColor: defaultPrimaryColor,
+    isDarkModeEnabled: true,
```
Comment (gao-sun) [inline packages/schemas/src/seeds/sign-in-experience.ts:13]: should be false per our discussion with guamian

File: packages/schemas/src/seeds/sign-in-experience.ts [hunk 2]
```diff
@@ -1,14 +1,17 @@
...
+    darkPrimaryColor: generateDarkColor(defaultPrimaryColor),
```
Comment (gao-sun) [inline packages/schemas/src/seeds/sign-in-experience.ts:14]: this should be empty if the above is false

**Review by gao-sun — APPROVED** [review]

File: packages/schemas/src/seeds/sign-in-experience.ts (hunk 1 above)
Comment (wangsijie) [reply packages/schemas/src/seeds/sign-in-experience.ts]: Yes, is a mistake on resolving conflict

File: packages/schemas/src/seeds/sign-in-experience.ts (hunk 2 above)
Comment (wangsijie) [reply packages/schemas/src/seeds/sign-in-experience.ts:14]: `darkPrimaryColor` is non-null, so need to generate a dark color even is disabled.

**Pull request merged**